src/components/index.ts

```diff
@@ -3,3 +3,4 @@ export {default as FlightData} from './FlightData/FlightData';
 export {default as FlightInfo} from './FlightInfo/FlightInfo';
 export {default as Header} from './Header/Header';
 export {default as Icon} from './Icon/Icon';
```

Comment:
To avoid the "cycles are allowed but..." warning, we either need to change the way we export Icon here or change the way we import it into the FlightInfo component.